conode/README.md

```diff
@@ -1,56 +1,612 @@
+```
+
+Otherwise, you can build it from the sources if you have the latest version of
+go:
+
+```
+$ git clone https://github.com/dedis/cothority.git
+$ cd conode
+$ go install ./conode
+```
+
+Then, you can launch the interactive setup configuration by running 
+
+```bash
+$ conode setup
+``` 
+
+Once done, the configuration files (the public.toml and private.toml) are saved
+in the default location or in the one specified during the interactive setup.
+
+The default locations are the following depending on the operating system:
+
+- Linux: `$HOME/.config/conode`
+- MacOS: `$HOME/Library/Application Support/conode`
+- Windows:`%AppData%\Conode`
+
+### Option 2: :whale: Configuration setup with docker
+
+If you have docker installed and running on your host, you can fire the
+interactive setup with the following command:
+
+```
+$ mkdir ~/conode_data
+$ docker run -it --rm -p 7770-7771:7770-7771 --name conode -v ~/conode_data:/conode_data dedis/conode:latest ./conode setup
+```
+
+This will prompt the interactive setup and set the 2 configuration files in the
+`~/conode_data` directory.
+
+### Post setup instructions
+
+There are a few things you can do now that you have your configuration files.
+This first one is to update the `Url` field in the public.toml file in case you
+set up a WebSocket over TLS connection for the conode-client communication
+during the [Environment setup](#environment-setup).
+
+Another option concerns the conode-conode communication that runs on TLS. If you
+would like the conode to run TLS on the WebSocket interface, you can tell it
+where to find the private key and a certificate for that key in the
+`private.toml` file with the following fields:
+
+```
+WebSocketTLSCertificate = "/etc/fullchain.pem"
+WebSocketTLSCertificateKey = "/etc/privkey.pem"
+```
+
+In this case, it is up to you to get get a certificate from a certificate
+authority and to update `fullchain.pem` when needed to renew the certificate.
+
+Using the Let's Encrypt CA and the `certbot` client program, you can get free
+certificates for domains which you control. `certbot` writes the files it
+creates into `/etc/letsencrypt`.
+
+If the user you use to run the conode has the rights to read from the directory
+where Let's Encrypt writes the private key and the current certificate, you can
+arrange for conode to share the TLS certificate used by the server as a whole:
+
+```
+WebSocketTLSCertificate = "/etc/letsencrypt/live/conode.example.com/fullchain.pem"
+WebSocketTLSCertificateKey = "/etc/letsencrypt/live/conode.example.com/privkey.pem"
+```
+
+Let's Encrypt certificates expire every 90 days, so you will need
+to restart your conode when the `fullchain.pem` file is refreshed.
+
+## Run your conode
+
+Once the setup is done with one of the two options, you can finally run your
+conode depending on the previously chosen option.
+
+### Option 1: :computer: Run with the command line
+
+Simply launch the following:
+
+```bash
+$ conode server
```

Comment:
we often run the command like `conode server | tee logfile.txt` so maybe it's a good idea to mention that the log files are not written anything so the user should use `tee` if they want to keep old logs